src/app/modules/startup/controller.nim

```diff
@@ -349,22 +347,46 @@ proc tryToObtainDataFromKeychain*(self: Controller) =
   let selectedAccount = self.getSelectedLoginAccount()
   self.keychainService.tryToObtainData(selectedAccount.keyUid)
 
+# FIXME: Remove with new login endpoint
```

Comment:
Is this fixed now?

Reply:
no, unfortunately, waiting for https://github.com/status-im/status-go/issues/4977.
Will write that explicitly in the comment